src/tools/update-fixtures.ts

```diff
@@ -52,7 +52,11 @@ async function updateFixture(options: UpdateFixtureOptions) {
   await exec('bower install', {cwd: sourceDir});
 
   // We're going to do an in-place conversion.
-  await fs.copy(sourceDir, convertedDir);
+  await fs.emptyDir(convertedDir);
+  await fs.copy(sourceDir, convertedDir, {
```

Comment:
If we're doing an in-place conversion, we need the bower components in the `convertedDir` so that they're properly analyzed as a part of conversion. 

Are we sure we're doing an in-place conversion here? A few lines down I see:
```
await convertPackage({
   inDir: sourceDir,
  outDir: convertedDir,
```

Which looks like we're still reading one directory and writing to another.

Reply:
> If we're doing an in-place conversion, we need the bower components in the convertedDir so that they're properly analyzed as a part of conversion.

Done. Now I include bower_components, but I exclude them during diffing.

> Are we sure we're doing an in-place conversion here? A few lines down I see:
> 
> await convertPackage({
>    inDir: sourceDir,
>   outDir: convertedDir,
> Which looks like we're still reading one directory and writing to another.

Oops, yeah. I guess it worked anyway, but now it's a real in-place upgrade.